packages/config/src/validate-config.ts

```diff
@@ -0,0 +1,250 @@
+import { logger } from '@pandacss/logger'
```

Comment:
Please add tests for all the different scenarios this function covers.

I would also recommend splitting them into dedicated functions so they can be isolated:

- validateTokens
- validateRecipes
- validateBreakpoints
- validateTokenReferences

etc.

Then we can compose them together in this function.